unstructured/ingest/cli/cmds/elasticsearch.py

```diff
@@ -80,9 +82,52 @@ def get_cli_options() -> t.List[click.Option]:
         return options
 
 
+@dataclass
+class ElasticsearchCliWriteConfig(CliConfig):
```

Comment:
I think we are importing ElasticsearchWriteConfig here now and don't have to declare the batch_size_bytes and num_processes attributes.

Reply:
[remove kwargs, inherit fields, add num-processes for upload to docs](https://github.com/Unstructured-IO/unstructured/pull/2152/commits/04fe747b437792ae68705a6ceed9970f9d6c221b)